src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts

```diff
@@ -217,6 +218,11 @@ export class HuiStatisticsGraphCardEditor
               required: false,
               selector: { boolean: {} },
             },
+            {
+              name: "energy_date_selection",
+              required: false,
+              selector: { boolean: {} },
+            },
```

Comment:
I feel like this option will cause a lot of confusion and should not be in the UI editor.

If we would put it in the UI editor, you should at least disable the `period` & `days_to_show` fields ones this option is enabled.

Reply:
Reverted in 17d20efe731a111249a76ee343fd31f7e24ffd99